MFaaP.MFWSClient.Tests/MFWSVaultValueListItemOperations.cs

```diff
@@ -149,6 +149,41 @@ public void GetValueListItems_WithLimitAndNameFilter()
 			runner.Verify();
 		}
 
+		/// <summary>
+		/// Ensures that a call to <see cref="MFaaP.MFWSClient.MFWSVaultValueListItemOperations.CreateValueListItemAsync"/>
+		/// requests the correct resource using the correct method.
+		/// </summary>
+		[TestMethod]
+		public async Task  CreateValueListItemAsync()
+        {
+			// Create our test runner.
+			var runner = new RestApiTestRunner<ValueListItem>(Method.POST, "/REST/valuelists/1/items");
+
+			// Execute.
+			await runner.MFWSClient.ValueListItemOperations.CreateValueListItemAsync(1, "new valuelistItem name");
+
+			// Verify.
+			runner.Verify();
+        }
+
+		/// <summary>
+		/// Ensures that a call to <see cref="MFaaP.MFWSClient.MFWSVaultValueListItemOperations.CreateValueListItem"/>
+		/// requests the correct resource using the correct method.
+		/// </summary>
+		[TestMethod]
+		public void CreateValueListItem()
```

Comment:
Please update the test method to be named "AddValueListItem", as per the name change listed below.